### PR TITLE
[cxx-interop] Always import cxx shim when cxx-interop is enabled

### PR DESCRIPTION
**Pull request by Huddie**

### CODE REVIEW

File: lib/Frontend/Frontend.cpp [hunk 1]
```diff
@@ -915,6 +923,10 @@ ImplicitImportInfo CompilerInstance::getImplicitImportInfo() const {
     }
   }
 
+  if(Invocation.getLangOptions().EnableCXXInterop && canImportCxxShim()) {
```
Comment (egorzhdan) [inline lib/Frontend/Frontend.cpp:926]: Please run clang-format

File: include/swift/Strings.h [hunk 2]
```diff
@@ -30,6 +30,8 @@ constexpr static const StringLiteral SWIFT_DISTRIBUTED_NAME = "Distributed";
 constexpr static const StringLiteral SWIFT_STRING_PROCESSING_NAME = "_StringProcessing";
 /// The name of the SwiftShims module, which contains private stdlib decls.
 constexpr static const StringLiteral SWIFT_SHIMS_NAME = "SwiftShims";
+/// The name of the CxxShim module, which contains cxx stdlib imports.
```
Comment (egorzhdan) [inline include/swift/Strings.h:33]: It actually contains a casting utility (see `swift/stdlib/public/Cxx/cxxshim/libcxxshim.h`), can you please update the comment?

**Review by egorzhdan — COMMENTED** [review]
Thanks!

Could you please also squash the commits into one?

**Review by zoecarver — APPROVED** [review]
Thank you!

**Comment by Huddie** [comment]
@swift-ci please smoke test

**Comment by Huddie** [comment]
@swift-ci please smoke test

**Comment by Huddie** [comment]
@swift-ci please smoke test

**Review by zoecarver — APPROVED** [review]
Thank you!

**Pull request merged**